client-app/src/examples/recalls/RecallsPanelModel.js

```diff
@@ -104,7 +104,7 @@ export class RecallsPanelModel {
         this.addReaction({
             track: () => this.searchQuery,
             run: () => this.loadAsync(),
-            delay: ONE_SECOND
+            debounce: ONE_SECOND
```

Comment:
Agree this would be a case where we wouldn't want to use the shorter/leading default I proposed in https://github.com/xh/hoist-react/pull/1680/files#r381927603 - where we are reacting at the keystroke level to issue a query back to a remote API. One second seemed long at first, but I think it's correct given how fast people might/might not type.